src/notation/internal/notationinteraction.h

```diff
@@ -50,6 +52,7 @@ class NotationInteraction : public INotationInteraction, public async::Asyncable
 {
     INJECT(notation, INotationConfiguration, configuration)
     INJECT(notation, ISelectInstrumentsScenario, selectInstrumentScenario)
+    INJECT(notation, playback::IPlaybackController, playbackController)
```

Comment:
You should not inject this interface here. It would be better if INotationInteraction::textEditingEnded sent the edited element. We can use this element to play it

Reply:
Thanks, done!